ci/test-bench.sh

```diff
@@ -40,7 +40,10 @@ fi
 BENCH_FILE=bench_output.log
 BENCH_ARTIFACT=current_bench_results.log
 
-# First remove "BENCH_FILE", if it exists so that the following commands can append
+# Ensure all dependencies are built
```

Comment:
@mvines You ok with this change to co?

Reply:
1. isn't `--all` at the top-level now implicit?
2. why `--release` when we don't use `--release` for the bench runs (or is `--release` implict there already)?